src/HttpSession.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021, Kung-Yao Hsieh, Shang-Chen LIN, Yi-Hang Xie. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef HTTPLIB_SRC_HTTPSESSION_H_	
+#define HTTPLIB_SRC_HTTPSESSION_H_	
+
+#include <string>
+
+#include "SocketSession.h"
+#include "HttpUtility.h"
+class SocketSession;
+
+/** @brief Handle the incoming http request. */
+class HttpSession {
+private:
+    /** @brief socket fd.*/
+    int socket_;
+    /** @brief Receive stream package.
+     *  @param msg Stream package's content.
+     */
+    void receiveMessage(const std::string &msg);
+    /** @brief Handle HttpRequest.
+     *  @param request Decoded HttpRequest.
+     */
+    void handleRequest(const HttpUtility::Request &request);
```

Comment:
Please add comment on this is called by receiveMessage() after the entire http request has been received.

Reply:
Done